pkg/spdk_test.go

```diff
@@ -326,7 +326,7 @@ func (s *TestSuite) TestSPDKMultipleThread(c *C) {
 			c.Assert(cksumBefore1, Not(Equals), "")
 
 			snapshotName1 := "snap1"
-			_, err = spdkCli.EngineSnapshotCreate(engineName, snapshotName1)
+			_, err = spdkCli.EngineSnapshotCreate(engineName, snapshotName1, nil)
```

Comment:
It's better to simply test this label feature in one of the unit tests

Reply:
As mentioned above https://github.com/longhorn/longhorn-spdk-engine/pull/343#discussion_r2039142500, actually we can't retrieve general labels, some extra development need to be done.

Reply:
OK. You can test if we can set and retrieve labels with specific keys first. Or add the testing after the extra API is delivered.